checker/src/org/checkerframework/checker/upperbound/UpperBoundVisitor.java

```diff
@@ -43,15 +70,63 @@ public Void visitArrayAccess(ArrayAccessTree tree, Void type) {
         // Is indexType LTL/LTOM of a set containing arrName?
         if ((indexType.hasAnnotation(LTLengthOf.class)
                         || indexType.hasAnnotation(LTOMLengthOf.class))
-                && (UpperBoundUtils.hasValue(indexType, arrName))) {
+                && (UpperBoundUtils.hasValue(indexType, arrName)
```

Comment:
Oh never mind I can't read. Also I put this comment on the wrong line...I thought the next line said arrName.toString but it says arrTree.toString. Disregard my mad rambling.